dataclients/kubernetes/endpointslices.go

```diff
@@ -74,20 +77,13 @@ func (eps *skipperEndpointSlice) targetsByServiceTarget(protocol, scheme string,
 	return result
 }
 
-func (eps *skipperEndpointSlice) targets(protocol, scheme string) []string {
-	result := make([]string, 0, len(eps.Endpoints))
-
-	var port int
-	for _, p := range eps.Ports {
-		if p.Protocol == protocol {
```

Comment:
this could mix STCP, UDP and TCP (that's iirc the protocol meaning in kubernetes)

Reply:
Decided to return addresses only